### DISCUSSION TITLE
How to reproduce the benchmark result?

### DISCUSSION THREAD

**Discussion by QuarticCat** [rui314/mold#742] [Q&A] [answered]
I'd like to try if I can do some optimizations or not. Is there any instruction on how to conduct a benchmark as you show in the README?

**Comment by rui314** [comment, marked as answer]
Just build programs with a verbose option (which depends on build system) to see what is actual linker command line arguments. Then invoke the linker with the same arguments with `time`.

**Comment by davidbolvansky** [comment]
Any data regarding memory usage? vs lld or gold?

I tried 1.6.0 on our project (private; distcc + ccache; mold -run make -j72) and near the end of build process it failed “mmap: failed”.
lld works fine.

**Reply by sicherha** [reply]
I have made a bunch of comparative measurements, including memory use, on a mid-sized developer laptop. You can find the results here: https://github.com/sicherha/linker-benchmarks/

**Reply by davidbolvansky** [reply]
Thanks, so slightly worse than lld.

**Comment by rui314** [comment]
Can you copy-n-paste the error message?
